pkgs/development/python-modules/soco/default.nix

```
@@ -16,7 +16,7 @@
 
 buildPythonPackage rec {
   pname = "soco";
-  version = "0.30.2";
+  version = "0.30.3";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and add `setuptools` to `build-system`